cfy_manager/components/nginx/config/http-external-rest-server.cloudify

```diff
@@ -2,6 +2,7 @@
 server {
   # server listening
   listen              *:80;
+  listen              [::]:80;
```

Comment:
Do we need to specify both? Most applications will listen on both if you only specify the IPv6 (::) one.